src/shared/utilities/textUtilities.ts

```diff
@@ -46,7 +46,7 @@ export function addCodiconToString(codiconName: string, text: string): string {
  * @returns Final output without any new lines or comments
  */
 export function stripNewLinesAndComments(text: string): string {
-    const blockCommentRegExp = /\/\*[.*?]\*\//
+    const blockCommentRegExp = /\/\*.*\*\//
```

Comment:
If this is multiline it's pretty expensive. We only use this in tests currently, right?